### DISCUSSION TITLE
adding a wms layer to the geometry preview window

### DISCUSSION THREAD

**Discussion by daamiansz** [dbeaver/dbeaver#17653] [Q&A] [answered]
Is it possible to add a wms service in the geometry preview window. I know it is possible to add a TMS layer, but can I add wms as in the example from leaflet ?

var nexrad = L.tileLayer.wms("http://mesonet.agron.iastate.edu/cgi-bin/wms/nexrad/n0r.cgi", {
layers: 'nexrad-n0r-900913',
format: 'image/png',
transparent: true,
attribution: "Weather data © 2012 IEM Nexrad"
});

**Comment by mkgrgis** [comment, marked as answer]
It depends on is there JavaScript "include" for L.wms or similar module. I haven't seen this include in web+JS map application for DBeaver.
